### PR TITLE
Be more vendor neutral.

### PR DESCRIPTION
**Pull request by lukemarsden**
Weave is not "recommended" over any other networks.

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1572)

### CODE REVIEW

**Comment by caseydavenport** [comment]
one comment, then LGTM.

@lukemarsden Looks like you also need the new linux foundation CLA.

**Comment by lukemarsden** [comment]
Thanks @caseydavenport, I've addressed the review feedback, rebased and squashed, and signed the LF CLA.

File: docs/getting-started-guides/kubeadm.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ To initialize the master, pick one of the machines you previously installed `kub
 **Note:** this will autodetect the network interface to advertise the master on as the interface with the default gateway.
 If you want to use a different interface, specify `--api-advertise-addresses=<ip-address>` argument to `kubeadm init`.
 
-If you want to use [flannel](https://github.com/coreos/flannel) as the pod network; specify `--pod-network-cidr=10.244.0.0/16` if you're using the daemonset manifest below. _However, please note that this is not required for any other networks, including Weave, which is the recommended pod network._
+If you want to use [flannel](https://github.com/coreos/flannel) as the pod network; specify `--pod-network-cidr=10.244.0.0/16` if you're using the daemonset manifest below. _However, please note that this is not required for any other networks._
```
Comment (devin-donnelly) [inline docs/getting-started-guides/kubeadm.md]: replace semicolon with comma; change last sentence to "However, please note that this is not required for any other networks besides Flannel."

**Comment by caseydavenport** [comment]
@lukemarsden any update on this guy?

**Comment by caseydavenport** [comment]
@lukemarsden looks like this needs a rebase.

**Pull request merged**